### DISCUSSION TITLE
running on SKR Mini E3 V3 w/o BTT TFT

### DISCUSSION THREAD

**Discussion by chiboreache** [luc-github/ESP3D#1025] [Questions]
is it possible?

**Comment by luc-github** [comment]
https://esp3d.io/esp3d/v3.x/hardware/system_boards/main_boards/btt_skr_mini_e3/index.html

**Reply by chiboreache** [reply]
my bad, diff are mostly cosmetics 

![image](https://github.com/luc-github/ESP3D/assets/16459251/b19990be-39a4-43e4-b03f-a34c317d2664)
